### PR TITLE
feat(8021x): functionality for wired configuration

### PR DESCRIPTION
**Pull request by tim-shockley**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: cypress/e2e/integration/profile/create.spec.ts [hunk 1]
```diff
@@ -92,4 +97,8 @@ describe('Test Profile Page', () => {
       })
     })
   })
+
+  it('should create profile with 8021x', () => {
+
```
Comment (madhavilosetty-intel) [inline cypress/e2e/integration/profile/create.spec.ts:102]: remove empty test

Comment (tim-shockley) [reply cypress/e2e/integration/profile/create.spec.ts]: done

File: src/app/core/navbar/navbar.component.html [hunk 2]
```diff
@@ -7,6 +7,11 @@
     <a mat-list-item routerLink="/" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
         <mat-icon>home</mat-icon><p data-cy="Dashboard"></p><span i18n="navlink">Dashboard</span>
     </a>
+    <a mat-list-item routerLink="/ieee8021x" routerLinkActive="active">
+        <mat-icon>security</mat-icon>
+        <span i18n="navlink">IEEE 8021.x</span>
```
Comment (bwendlandt-intel) [inline src/app/core/navbar/navbar.component.html:12]: Small typo '8021.x', I'd make a PR but didn't want to mess up whatever you got going on.

**Comment by tim-shockley** [comment]
Fixed and pushed, thanks

File: src/app/core/navbar/navbar.component.html (hunk 2 above)
Comment (madhavilosetty-intel) [reply src/app/core/navbar/navbar.component.html]: resolved

**Review by madhavilosetty-intel — APPROVED** [review]

**Pull request merged**